roles/cfg_openwrt/tasks/render_etc.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Create /etc directories part one
```

Comment:
would be great have a better naming than part one and part two. e.g. (static files, templated files)